src/stateless/selectbox.vue

```diff
@@ -237,6 +255,16 @@ export default {
         setSearch: debounce(function (v) {
             this.searchTextValue = v
         }, 400),
+        loadAsyncOptions () {
+            if (typeof this.getOptions === 'function') {
+                this.isSearchLoading = true
+                this.getOptions(this.searchText || '')
+                    .then(options => {
+                        this.isSearchLoading = false
+                        this.asyncOptions = options
+                    }).catch(console.error)
```

Comment:
We should probably catch error and log that error

Reply:
What do you mean? That's what it does.

Reply:
:+1:

Reply:
Maybe it would make sense to also clear loading if search fails?